code/modules/clothing/under/jobs/Plasmaman/civilian_service.dm

```diff
@@ -19,7 +19,7 @@
 
 /obj/item/clothing/under/plasmaman/enviroslacks
 	name = "enviroslacks"
-	desc = "The pet project of a particularly posh plasmaman, this custom suit was quickly appropriated by Nano-Trasen for it's detectives, lawyers, and bar-tenders alike."
+	desc = "The pet project of a particularly posh plasmaman, this custom suit was quickly appropriated by Nano-Trasen for its detectives, lawyers, and bar-tenders alike."
```

Comment:
```suggestion
	desc = "The pet project of a particularly posh plasmaman, this custom suit was quickly appropriated by Nanotrasen for its detectives, lawyers, and bartenders alike."
```

Reply:
Another instance of Nano-Transen is located in `code/modules/clothing/spacesuits/plasmamen.dm` while we're all thinking about it.

Reply:
Found that one plus a whole bunch more issues in that commit